test/get-last-release.test.js

```diff
@@ -43,6 +43,31 @@ test('Get the highest prerelease valid tag, ignoring other tags from other prere
   });
 });
 
+test('Get the hightest prerelease valid tag, when other tags share the same HEAD', (t) => {
```

Comment:
this test name doesnt make the goal clear to me. semantic-release doesnt really consider the relationship between typical pre-release stages, so if "highest" is referring to promotion through those stages, i don't think that is worth highlighting here. the real goal is to ensure that the pre-release version is properly with the pre-release defined for that branch, correct?

Reply:
Yeah, that's correct. The goal is more to ensure that the pre-release version corresponds with the pre-release for the branch indeed. I'll change it out to reflect that!